app/api/v1/endpoints/chat_session_endpoints.py

```
@@ -266,6 +290,12 @@
             "has_next": offset + request.page_size < total_count,
             "has_previous": request.page > 1
         }
+        
+    except HTTPException:
+        raise
+    except Exception as e:
+        logger.error(f"Error listing chat messages: {e}")
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)